src/rp2_common/boot_bootrom_headers/include/boot/bootrom_constants.h

```diff
@@ -304,37 +308,37 @@ typedef struct cflash_flags {
     uint32_t flags;
 } cflash_flags_t;
 
+#endif
```

Comment:
Might be worth a quick comment here that this relates to the `#ifndef __ASSEMBLER__`? Or maybe move the `CFLASH` stuff above that so that the assembler-only stuff is all in one place at the top?